appendix-basic65.tex

```diff
@@ -220,168 +252,240 @@ \subsection{BASIC 65 Variables}
 \end{tabular}
 \end{center}
 
+A variable whose name is a single letter followed by the type symbol (or no
+symbol for real number variables) is a {\em fast variable}. BASIC 65 stores
+the variable in a way that makes it faster to access or update the value than
+variables with longer names. It otherwise behaves like any other variable.
+
 \subsection{BASIC 65 Arrays}
 \index{BASIC 65 Arrays}
 
-Each array consumes the number of elements multiplied by the item size,
```

Comment:
Old text gave a multi-dimension example and provided internal details on where things were stored and how much space it'd consume

So I'm guessing you wanted to minimise surface of the api with this too

Yeah, I guess I'm ok with that, maybe these things could change in future